api/tests/unit/application/courses/index_test.js

```diff
@@ -79,4 +82,32 @@ describe('Unit | Router | course-router', function() {
     });
   });
 
+  describe('POST /api/courses', function() {
+
+    let sandbox;
+
+    before(() => {
+      sandbox = sinon.sandbox.create();
+
+      sandbox.stub(connectedUserVerification, 'verifyByToken').callsFake((request, reply) => reply('decodedToken'));
+      sandbox.stub(accessSessionHandler, 'sessionIsOpened').callsFake((request, reply) => reply('decodedToken'));
+      sandbox.stub(CourseController, 'save').callsFake((request, reply) => reply('ok'));
+    });
+
+    after(() => {
+      sandbox.restore();
+    });
+
+    it('should exist', (done) => {
+      expectRouteToExist({ method: 'POST', url: '/api/courses' }, done);
+    });
+
+    it('should call pre-handler', (done) => {
```

Comment:
Pourrait-on le renommer afin que ça corresponde mieux au comportement ?
Du genre "should check if user is connected and session opened"

Reply:
C'est vrai, j'ai fait la modif.